controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/horizon/HorizonClient.java

```diff
@@ -9,7 +9,7 @@
  */
 public interface HorizonClient {
 
-    InputStream getMetrics(Slime query);
+    InputStream getMetrics(byte[] query);
```

Comment:
Not possible to overwrite/replace/remove object with Slime, so had to do it with Jackson...